Fabric.Identity.API/scripts/tests/CatalystDosIdentity.functional.tests.ps1

```diff
@@ -90,31 +90,365 @@ Describe 'Identity Cli Functional Tests' {
             }
         }
     }
-    
-    Describe 'Get-ClientRegistration' {}
-        
-    Describe 'New-ClientRegistration' {}
-    
-    Describe 'New-ImplicitClientRegistration' {}
-    
-    Describe 'New-HybridClientRegistration' {}
-    
-    Describe 'New-HybridPkceClientRegistration' {}
-    
-    Describe 'Invoke-UpdateClientRegistration' {}
-    
-    Describe 'Invoke-UpdateClientPassword' {}
-    
-    Describe 'Test-IsClientRegistered' {}
-    
+
+    Describe 'Client Flows' {
+        Context 'Happy Client Lifecycle' {
+            It 'Test, create, update, get, recreate, reset, test' {
```

Comment:
Just a thought, what do you think of the idea of making the initial 

> It 'Test, create, update, get, recreate, reset, test'

into a context instead and having each comment in the test become an 'It' block instead?

That way if stuff fails we can narrow it down easier as well

Reply:
good call.  The it statements do not denote scope right?  so the variables carry through?

Reply:
I believe they will pass variables through